tasks/collectd.yml

```diff
@@ -0,0 +1,23 @@
+# tasks file for influxdb
+---
+- name: collectd | create collectd_auth_file
+  template:
+    src: etc/influxdb/collectd_auth_file.j2
+    dest: /etc/influxdb/collectd_auth_file
+    owner: "{{ item.owner | default('root') }}"
+    group: "{{ item.group | default('root') }}"
+    mode: "{{ item.mode | default('0644') }}"
+  when: influxdb_collectd_users | default([]) | length > 0
+  notify: restart influxdb
+  tags:
+    - influxdb-collectd-create
+    - influxdb-collectd-create-auth-file
+
+- name: collectd | remove collectd_auth_file
+  file:
+    path: /etc/influxdb/collectd_auth_file
```

Comment:
Can you put this in a variable like this?

https://github.com/Oefenweb/ansible-apt/blob/master/vars/main.yml#L3

Reply:
And then use it like this?

https://github.com/Oefenweb/ansible-apt/blob/master/tasks/main.yml#L19